jobs/gradle.yml

```diff
@@ -0,0 +1,50 @@
+# gradle
+#
+# Job responsible for building Gradle Wrapper based projects.
+#
+
+parameters:
+  jdkVersions:
+    - name: JDK_8
+      version: "1.8"
+    - name: JDK_11
+      version: "1.11"
+  jobs:
+    - name: Ubuntu_16
+      vmImage: ubuntu-16.04
```

Comment:
Ubuntu version impacts Gradle build much less then version of Java. The build will run in Java virtual machine which work the same on any Ubuntu, from Gradle build's point of view

Reply:
In fact that would mean we don't need to run it on macOS and Windows, except the JVM implementation differs on those platforms

Reply:
It is partially true. File operations might still hit people here, especially with older versions of Gradle used. But also old versions will fail on JDK 11 due to lack of support. Those who converted their scripts to Kotlin are safe here too. For simple projects I think we can test only on one platform indeed.